runtime/src/asset.rs

```diff
@@ -582,14 +589,29 @@ decl_module! {
                 // New total supply must be valid
                 token.total_supply = updated_total_supply;
             }
-
-            // After checks are ensured introduce side effects
+            let round = Self::funding_round(ticker.clone());
```

Comment:
We can pass a reference here rather than a clone.`Self::funding_round(ticker.clone())` should work fine.